dotfyle

```diff
@@ -159,6 +159,7 @@ update() {
   repos=$(find_git "$location")
   for repo in $repos; do
     echo "Updating $repo"
+    git -C "$location/$repo" reset --hard HEAD >/dev/null
```

Comment:
It's a bit scary with a hard reset without a warning, I often keep a lot of uncommited code. Maybe consider stashing and unstashing.

Reply:
Do you see that being a use-case of this? If so then `--autostash` would be the better option.
I want to try to avoid the chance of merge conflicts if possible since this is originally intended to be a try out something new

Reply:
That might create conflicts when remote changes are merged for instances where the user changes have diverged.

I would just have a confirm dialog to accept the changes from remote, so that the user can then stash manually and then merge them cleanly.